unattended_installer/builder.sh

```diff
@@ -265,4 +268,26 @@ function builder_main() {
     fi
 }
 
+function checkFilebeatURL() {
+
+    # Import variables
+    . ${resources_installer}/installVariables.sh
+    new_filebeat_url="https://raw.githubusercontent.com/wazuh/wazuh/master/extensions/elasticsearch/7.x/wazuh-template.json"
```

Comment:
Instead of loading the entire `installVariables.sh` script, get the value you want with the `grep` command and store it in the `new_filebeat_url` variable.

Reply:
Solved issues caused by this in this commit https://github.com/wazuh/wazuh-packages/pull/2001/commits/8b98b000428fd6efebb97e3a79ee2146ff022cef

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/2001/commits/e2be2087a81b462bd4c9f30b83c72ba6ce6f66a1